pdns/pdnsutil.cc

```diff
@@ -598,6 +598,27 @@ int checkZone(DNSSECKeeper &dk, UeberBackend &B, const DNSName& zone, const vect
     }
   }
 
+  std::map<std::string, std::vector<std::string>> metadatas;
+  if (!B.getAllDomainMetadata(zone, metadatas)) {
```

Comment:
For terrible reasons, this skips `PRESIGNED`, `NSEC3PARAM`, and `NSEC3NARROW`for gsqlbackend (but not for other backends) - but I don't think we can blame this PR for that!

Reply:
How best to address the issue?

Reply:
So based on the code it seems that the only case where they are not included is if dnssec is disabled on the backend. Do we really care about the rare case where dnssec is disabled and there are DNSSEC params?

Reply:
I've decided we don't :)